src/components/DAO/DAO.tsx

```diff
@@ -1,22 +1,540 @@
+          functionName: "balanceOf",
+          args: [contractData[0][6]],
+        });
+
+        //tvl
+        const totalTvl: bigint = $vaults[6].reduce(
+          (total: bigint, numero: bigint) => total + numero,
+          BigInt(0)
+        );
+        const _totalTvl = formatUnits(totalTvl, 18);
+
+        //profit Token
+        const _profitTotalSupply = formatUnits(profitTotalSupply, 18);
+        const _profitTokenPrice = Number(totalTvl) / Number(profitTotalSupply);
+        const profitMarketCap =
+          Number(_profitTokenPrice) * Number(_profitTotalSupply);
+
+        const profitToken: ProfitTokenData = {
+          price: _profitTokenPrice,
+          totalSupply: _profitTotalSupply,
+          marketCap: profitMarketCap,
+        };
+
+        setProfitTokenData(profitToken);
+
+        //sdiv token
+        setSdivTotalSupply(formatUnits(_sdivTotalSupply, 18));
+
+        //pm
+        setPmTotalSupply(formatUnits(_pmTotalSupply, 18));
+
+        //platformData
+
+        const percentageFees: string[] = platformFees.map((fee: bigint) =>
+          fee !== 0n ? (fee / 1000n).toString() + " %" : "0 %"
+        );
+
+        //treasury
+        const _treasuryBalance = Number(
+          formatUnits(treasuryBalance, 18)
+        ).toFixed(2);
+
+        //team
+        const _multisig = Number(formatUnits(multisigBalance, 18)).toFixed(2);
+
+        const platformData: PlatformData = {
+          platformVersion: platformVersion,
+          platformGovernance: contractData[0][5],
+          multisig: contractData[0][6],
+          multisigBalance: _multisig,
+          numberOfTotalVaults: $balances[3].length,
+          totalTvl: _totalTvl,
+          strategieNames: contractData[6],
+          platformFee: percentageFees[0],
+          vaultManagerFee: percentageFees[1],
+          strategyLogicFee: percentageFees[2],
+          ecosystemFee: percentageFees[3],
+          treasuryBalance: _treasuryBalance,
+        };
+        setPlatformData(platformData);
+      } catch (error) {
+        console.error("Error fetching platform data:", error);
+      }
+    }
+  };
+
   return (
-    <div className="dao pt-2">
-      <h1 className="text-xxl text-gradient mb-3">Platform</h1>
-      <div></div>
-      <br />
-
-      <h1 className="text-xxl text-gradient mb-3">Tokenomics</h1>
-      <div></div>
-      <br />
-
-      <h1 className="text-xxl text-gradient mb-3">Governance</h1>
-      <div></div>
-      <br />
-
-      <h1 className="text-xxl text-gradient mb-3">Team</h1>
-      <div></div>
-      <br />
-    </div>
+    <main className="p-0 w-full m-auto">
+      <section className="w-4/5 m-auto">
+        <article className="m-auto p-2 bg-button rounded-md w-full">
+          <h1 className="text-xxl text-gradient mb-3 text-left">Platform</h1>
+          <div className="grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3  gap-3 p-0">
+            <article className="m-0 p-2 rounded-md text-left grid bg-[#1c1c23]">
```

Comment:
Dont't use section/article/br etc. tags, better use div, astro himself under the hood converts this into the correct layout